verkle-trie/src/proof/stateless_updater.rs

```diff
@@ -525,10 +538,10 @@ impl SparseVerkleTree {
         // Now lets fetch the parent node's commitment and recursively update each parent
 
         while !prefix.is_empty() {
-            let child_index = prefix.pop().unwrap();
-            // If we have never updated the parent node before,
-            // then it will be the old commitment
-            // If we have then it will be in updated commitments
+            let child_index = prefix.pop().unwrap(); // Safety: Fine unwrap because we've checked prefix isn't empty
+                                                     // If we have never updated the parent node before,
```

Comment:
Why was the comment moved here?

Reply:
Not sure, some weird format thing, moved it back